docs/tutorial.rst

```diff
@@ -276,6 +276,67 @@ See the SLiM manual and/or
 `Urrichio & Hernandez (2014) <https://www.genetics.org/content/197/1/221.short>`__
 for more discussion.
 
+Simulating genomes with selection
+---------------------------------
+
+In the example above given a species, recombination map, and demography, a contig
+was simulated assuming strictly neutrality. It is possible to incorporate selection in the
+simulations by (1) specifying the Distribution of Fitness Effects (DFE) for all new mutations
+across the genome or in a contig; or by (2) adding a single mutation under selection,
+as for instance in a selective sweep.
+
+If a DFE is already described in the catalog, one can incorporate it into the simulation
+with the flag ``--dfe``. For instance, HomSap has a DFE described as Gamma_K17.
+To add it the example above we can type as follows:
+
+.. code-block:: console
+
+    $ stdpopsim -e slim --slim-scaling-factor 10 HomSap \
+    $    -c chr22 -l 0.05 --dfe Gamma_K17 -o foo.ts -d OutOfAfrica_2T12 2 4
+
+This example will simulate selection following the proportion of sites described in Gamma_K17.
+
+Instead of simulating the whole contig one can simulate only CDSs by using the flag ``--dfe-annotation``:
+
+.. code-block:: console
+
+    $ stdpopsim -e slim --slim-scaling-factor 10 HomSap \
+    $    -c chr22 -l 0.05 --dfe Gamma_K17 -o foo.ts -d OutOfAfrica_2T12 \
+    $    --dfe-annotation ensembl_havana_104_CDS 2 4
+
+If instead of an annotation file (such as a .gff) one has a bed file (ex.bed) like the one below:
+
+.. code-block:: console
+
+    $ chr1  100000  145000
+    $ chr1  150000  302425
+    $ chr1  150000  302425
```

Comment:
This is not something on the command line, so I think no `$` (which I anyhow take to mean "bash prompt")
```suggestion
     chr1  100000  145000
     chr1  150000  302425
     chr1  150000  302425
```